frontend/resourceadm/pages/PartyListPage/PartyListPage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { StudioSpinner } from '@studio/components';
+import { PartyListDetail } from 'resourceadm/components/PartyListDetails/PartyListDetail';
+import { useGetPartyListQuery } from 'resourceadm/hooks/queries/useGetPartyList';
+import { getPartyListPageUrl } from 'resourceadm/utils/urlUtils/urlUtils';
+
+export const PartyListPage = (): React.ReactNode => {
+  const { t } = useTranslation();
+
+  const { selectedContext, env, listId } = useParams();
+  const repo = `${selectedContext}-resources`;
```

Comment:
I believe you created a hook for this functionality the other day, named ```useUrlParams```. I think you should use that instead to get the selected context and repo 😄